README.md

```diff
@@ -1,38 +1,16 @@
+[![Latest Stable Version](https://poser.pugx.org/typo3/surf/v/stable.svg)](https://packagist.org/packages/TYPO3/Surf)
+[![Total Downloads](https://poser.pugx.org/typo3/surf/downloads.svg)](https://packagist.org/packages/TYPO3/Surf)
+[![Monthly Downloads](https://poser.pugx.org/typo3/surf/d/monthly)](https://packagist.org/packages/TYPO3/Surf)
 ![Tests](https://github.com/TYPO3/Surf/workflows/Tests/badge.svg?branch=master&event=push)
 [![Coverage Status](https://img.shields.io/coveralls/TYPO3/Surf/master.svg?style=flat-square)](https://coveralls.io/github/TYPO3/Surf?branch=master)
-[![Downloads](https://img.shields.io/packagist/dt/TYPO3/Surf.svg?style=flat-square)](https://packagist.org/packages/TYPO3/Surf)
 
-# Powerful and flexible deployment tool for PHP projects
+# Surf
```

Comment:
this is way too minimalistic. I'd rather even add more content here

Reply:
We use the extension/package/application name here in most cases, see standard.

Reply:
This title is super important for SEO reasons and the content is super important for people who browse through github.

Reply:
Maybe TYPO3 Surf would be even better? But i was not sure, if that fits for the Flow/Neos guys :)

Reply:
Ok.

Reply:
> This title is super important for SEO reasons and the content is super important for people who browse through github.

As for the importance of SEO, it seems that GitHub cares most and perhaps only about the _About_ field of the project and not at all about the README. For example, search for "deployment php" and find Surf on the first page of results, and then search for "deployment php project" and don't find Surf at least on the first 22 pages of results. In the results, GitHub highlights the search terms, and it seems to me that this is all about _About_.

As for the content: the idea is to give the reader and decision maker a quick overview with the abstract, and if they decide this is about what they were looking for, the full documentation is linked and quickly at hand, as is the store (packagist) to proceed with the installation. This is consistent with the logic in scientific papers, where the professor reviews papers, searches the abstract for relevant keywords, and then decides whether to actually read the paper. At the same time, the documentation maintainer only has to worry about one place for the documentation, Documentation/.